### PR TITLE
Cancel delete admin requests

### LINKED ISSUE #1658: Admin Cancel/Delete Requests
The admin feature for Deleting requests should be updated.

- Non-Completed requests should be Cancelled
- Completed requests should be Deleted

The buttons and menu should be updated to reflect this.

### PR DESCRIPTION
**Pull request by TheBurchLog**
Fixes: #1658 

image

### CODE REVIEW

File: src/app/beer_garden/requests.py [hunk 1]
```diff
@@ -881,6 +881,28 @@ def complete_request(
...
+    """
+    requests = db.query(Request, filter_params=kwargs)
+
+    for request in requests:
+        try:
+            cancel_request(request=request)
+        except RequestStatusTransitionError:
+            logger.error("Attempted to cancel an already completed Request")
+
+    logger.info(f"Cancled {len(requests)} requests")
```
Comment (1maple1) [inline src/app/beer_garden/requests.py:902]: spelling: Canceled

File: src/ui/src/templates/admin_request_delete.html [hunk 2]
```diff
@@ -12,89 +12,122 @@ <h3 class="modal-title" id="modal-title">
...
-        <button type="button" class="btn btn-warning btn-block" ng-click="deleteRequests('IN PROGRESS')"
-          confirm="Are you sure you want to delete Requests with status IN PROGRESS? There may be a plugin already running the request.">
-          Delete IN PROGRESS
+        <button type="button" class="btn btn-warning btn-block" ng-click="deleteCancelRequests('CREATED', is_cancel=true)"
+          confirm="Are you sure you want to cancel Requests with status CREATED? Recommend clearing topics as well.">
+          Cancel CREATED
+        </button>
+        <button type="button" class="btn btn-danger btn-block" ng-click="deleteCancelRequests('CREATED')"
+          confirm="Are you sure you want to delete Requests with status CREATED? Recommend clearing topics as well.">
+          Cancel CREATED
```
Comment (1maple1) [inline src/ui/src/templates/admin_request_delete.html]: Delete CREATED

**Review by 1maple1 — CHANGES_REQUESTED** [review]
Double check cancel vs delete.

**Comment by TheBurchLog** [comment]
UI Updates:
image

**Review by 1maple1 — APPROVED** [review]
Great, overall looks a bit more streamlined.

**Pull request merged**